### DISCUSSION TITLE
LevelDB 源码分析「一、基本数据结构」 | SF-Zhou's Blog

### DISCUSSION THREAD

**Discussion by giscus** [SF-Zhou/sf-zhou.github.io#199] [Announcements]
# LevelDB 源码分析「一、基本数据结构」 | SF-Zhou's Blog

https://sf-zhou.github.io/leveldb/leveldb_01_data_structure.html

**Comment by pengchenglv** [comment]
hash表的作用应该是在o(1)的时间内取出值，而不是维护used属性

**Comment by JadenWqy** [comment]
hashtable 的实现是单独链表法(LRUHanle chaining via next_hash)，而非开放式哈希 (open address via probing）
